internal/k8s/controllers/service_controller.go

```diff
@@ -47,8 +47,9 @@ type ServiceReconciler struct {
 	Endpoints         NeedEndPoints
 	LoadBalancerClass string
 	Reload            chan event.GenericEvent
-	// initialLoadPerformed turns true after the first reprocessAll. while false don't reconcile services
-	// because we care about loading the already assigned services first.
+	// initialLoadPerformed is set after the first time we call reprocessAll.
```

Comment:
this should belong to the previous commit

Reply:
fixed